exampleCourse/questions/element/matching/question.html

```diff
@@ -1,21 +1,113 @@
+    <pl-statement match="Mexico City">Mexico</pl-statement>
+    <pl-statement match="Paris">France</pl-statement>
+    
+    <pl-option>Tijuana</pl-option>
+    <pl-option>Toronto</pl-option>
+    <pl-option>New York City</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 2: <code>blank="true"</code> presents blank drop-down lists to begin,
+        and students receive Invalid, Not Gradeable errors if they leave any statement blank.
+    </p>
+</pl-question-panel>
+    
+<pl-matching answers-name="ex2" blank="true">
+    <pl-statement match="Ottawa">Canada</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
+    <pl-statement match="Mexico City">Mexico</pl-statement>
+    <pl-statement match="Paris">France</pl-statement>
+    
+    <pl-option>Tijuana</pl-option>
+    <pl-option>Toronto</pl-option>
+    <pl-option>New York City</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 3: <code>number-statements</code> and <code>number-options</code>
+        determine how many statements and options are shown. Statements are sampled at random.
+        The options will always include each correct match for the sampled statements,
+        plus random distractors.
     </p>
 </pl-question-panel>
     
-<pl-matching answers-name="capitals" number-statements="3" number-options="5" blank="true">
+<pl-matching answers-name="ex3" number-statements="3" number-options="5">
     <pl-statement match="Ottawa">Canada</pl-statement>
-    <pl-statement match="d.c.">United States</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
     <pl-statement match="Mexico City">Mexico</pl-statement>
     <pl-statement match="Paris">France</pl-statement>
     
-    <pl-option>Ottawa</pl-option>
-    <pl-option name="d.c.">Washington, D.C.</pl-option>
-    <pl-option>Mexico City</pl-option>
     <pl-option>Tijuana</pl-option>
-    <pl-option>Calgary</pl-option>
     <pl-option>Toronto</pl-option>
     <pl-option>New York City</pl-option>
-    <pl-option>Paris</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 4: if <code>number-options</code> is smaller than the amount of options
+        required to match the sampled statements, None of the Above will be added.
+    </p>
+</pl-question-panel>
+    
+<pl-matching answers-name="ex4" number-statements="3" number-options="2">
+    <pl-statement match="Ottawa">Canada</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
+    <pl-statement match="Mexico City">Mexico</pl-statement>
+    <pl-statement match="Paris">France</pl-statement>
+    
+    <pl-option>Tijuana</pl-option>
+    <pl-option>Toronto</pl-option>
+    <pl-option>New York City</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 5: the <code>counter-type</code> can be changed to lower-alpha, upper-alpha, or
+        decimal.
```

Comment:
```suggestion
        Example 5: the <code>counter-type</code> option indicates how options are identified. It can be set to <code>lower-alpha</code> (a, b, c, ...), <code>upper-alpha</code> (A, B, C, ...), or <code>decimal</code> (1, 2, 3).
```